components/compas-scl-lists/src/compas-scl-list.ts

```diff
@@ -1,54 +1,43 @@
-import {
-  css,
-  customElement,
-  html,
-  LitElement,
-  property,
-  PropertyValues,
-  state,
-  TemplateResult,
-} from 'lit-element';
-
-import '@material/mwc-icon';
-import '@material/mwc-list';
-import '@material/mwc-list/mwc-list-item';
-import '@openscd/oscd-filtered-list';
-import { translate } from 'lit-translate';
-
-import { SelectedItemsChangedEvent } from '../../to-be-made-a-dependency/oscd-filter-button.js';
-import '../../to-be-made-a-dependency/oscd-filter-button.js';
-import './compas-loading.js';
+import { css, html, LitElement, property, TemplateResult } from "lit-element";
+
+import "@material/mwc-icon";
+import "@material/mwc-list";
+import "@material/mwc-list/mwc-list-item";
+import "@openscd/oscd-filtered-list";
+
+import type { SelectedItemsChangedEvent } from "@openscd/oscd-filter-button";
+import "@openscd/oscd-filter-button";
+import "@com-pas/compas-loading";
 
 /* Event that will be used when a SCL is selected from a list of SCL Documents. */
 export interface SclSelectedDetail {
   docId: string;
 }
 export type SclSelectedEvent = CustomEvent<SclSelectedDetail>;
 export function newSclSelectedEvent(docId: string): SclSelectedEvent {
-  return new CustomEvent<SclSelectedDetail>('scl-selected', {
+  return new CustomEvent<SclSelectedDetail>("scl-selected", {
     bubbles: true,
     composed: true,
     detail: { docId },
   });
 }
 
-@customElement('compas-scl-list')
 export class CompasSclList extends LitElement {
```

Comment:
missing JSDoc here and for every prop like how it's done in `oscd-components` 
https://github.com/openscd/oscd-components/blob/main/components/oscd-action-icon/src/oscd-action-icon.ts

Reply:
Out of scope